WordPress/src/test/java/org/wordpress/android/viewmodel/giphy/provider/TenorProviderTest.kt

```diff
@@ -29,17 +29,22 @@ import org.wordpress.android.viewmodel.giphy.provider.GifProvider.GifRequestFail
 import org.wordpress.android.viewmodel.giphy.provider.TenorProviderTestFixtures.expectedGifMediaViewModelCollection
 import org.wordpress.android.viewmodel.giphy.provider.TenorProviderTestFixtures.mockedTenorResult
 import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 
+@ExperimentalCoroutinesApi
```

Comment:
I think you can remove this now 👍

Reply:
Thank you for point this out, it's fixed in 3d8826a3ae30e05cda34be8f5510da4d4d1f605f